suite-native/module-add-accounts/src/hooks/useAddCoinAccount.ts

```diff
@@ -40,8 +54,10 @@ export const useAddCoinAccount = () => {
     );
     const device = useSelector(selectDevice);
     const { showAlert, hideAlert } = useAlert();
-    const navigation =
-        useNavigation<StackNavigationProps<RootStackParamList, RootStackRoutes.AccountDetail>>();
+    const navigation = useNavigation<NavigationProps>();
+    const [networkWithTypeToAdd, setNetworkWithTypeToAdd] = useState<
+        [Network, AccountType] | undefined
+    >(undefined);
```

Comment:
It would be better to just use empty array for default value here IMO

```suggestion
    const [networkWithTypeToAdd, setNetworkWithTypeToAdd] = useState<
        [Network, AccountType]
    >([]);
```

Reply:
Is this state actually necessary? It might be possible to get rid of it completely?

Reply:
I'll think about this once more.

Reply:
Changing name to better address what is in it, but this feels being the best way to achieve what it should do.

Reply:
5c5f2af2c4